src/NetAnalyzers/UnitTests/Microsoft.CodeQuality.Analyzers/Maintainability/CodeMetricsComputationTests.cs

```diff
@@ -1900,6 +1878,28 @@ class C1
             VerifyCSharp(source, expectedMetricsText);
         }
 
+        [Fact, WorkItem(2133, "https://github.com/dotnet/roslyn-analyzers/issues/2133")]
+        public void MethodWithAnonymousType()
+        {
+            var source = @"
+class C
+{
+    object M1(C1 c)
```

Comment:
Why was the test moved and new parameter of an undefined type `C1` added? Can we revert this test change?

Reply:
I did the PR a while ago and it seems that I picked the wrong method during the merge. I am reverting this change.